rotary/rules/rotary.smk

```diff
@@ -221,17 +217,45 @@ rule build_gtdb_mash_ref_database:
         """
 
 
+rule set_up_sample_directories:
+    input:
+        "samples.tsv"
```

Comment:
I wonder if this would recognize the variable `sample_tsv_path` from above? If so, that might be an elegant solution to use that variable as the input rather than writing "samples.tsv" twice in the snakefile. `sample_tsv_path` might need to be changed to uppercase for clarity (since it would behave as a global variable) in that case

Reply:
Addressed